sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_shared/encryption.py

```diff
@@ -5,15 +5,19 @@
 # --------------------------------------------------------------------------
```

Comment:
The files in the `_shared` folder are supposed to match across all packages. It's our "common" I guess since Python does not have a common package. Why try not to introduce differences in these files across packages but I also agree I don't love having it here and in `file-share` as it can cause confusion when looking for what support encryption.

Post-Preview I need to do some cleanup of some references to encryption that were copied/pasted into datalake/file-share and I will consider removing these files shared all together at that time.